v2/index/index.go

```diff
@@ -61,7 +65,7 @@ type (
 	}
 
 	// IterableIndex extends Index in cases where the Index is able to
-	// provide an iterator for getting the list of all entries in the
+	// provide an iterator for getting the list of all multihashes in the
 	// index.
 	IterableIndex interface {
 		Index
```

Comment:
Here and a few lines down - probably need to clarify what happens when there are multiple offsets for the same MH...